IPs/RSKIP170.md

```diff
@@ -0,0 +1,71 @@
+# Peg-in to any address
+
+|RSKIP          |170           |
+| :------------ |:-------------|
+|**Title**      |Peg-in to any address |
+|**Created**    |01-SEP-20 |
+|**Author**     |MI |
+|**Purpose**    |USa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft |
+
+## Abstract
+
+This RSKIP proposes a way for users to be able to lock funds from BTC, indicating the RSK address in which they want their RBTCs transferred to and a BTC refund address in case the peg-in can not be completed for some reason.
+
+## Motivation
+
+Allowing users to lock funds to any address in RSK.
+
+## Specification
```

Comment:
I think you could give an idea on how this would be included in the existing flow:
1. Blockchain verifications (not modified)
2. parsing
2.1. rejection if parsing is invalid (following existing rules of refund)
2.2. flow if there is no op_return (not modified)
3. flow if op_return is present and valid

Reply:
Abstract could start by: _This RSKIP allows a user to transfer BTC to RSK_, to make it more clear